javascript-view-swapping/main.js

```diff
@@ -0,0 +1,27 @@
+var $tabContainer = document.querySelector('.tab-container');
+
+var $tabElements = document.querySelectorAll('.tab');
+var $viewElements = document.querySelectorAll('.view');
+
+function tabClicker(event) {
+  if (event.target.matches('.tab')) {
+    for (var i = 0; i < $tabElements.length; i++) {
+      if (event.target === $tabElements[i]) {
+        $tabElements[i].classList.add('active');
+        for (var j = 0; j < $tabElements.length; j++) {
```

Comment:
nothing wrong with this, but one loop would work with an else statement to reset all the classnames for the one which doesn't match the target.